README.md

```diff
@@ -41,13 +41,13 @@ git clone git@github.com:{YOUR_NAME_HERE}/quickstart-scala-sbt-native-image.g8.g
 cd quickstart-scala-sbt-native-image.g8
 git remote add upstream git@github.com:sbchapin/quickstart-scala-sbt-native-image.g8.git
 
-# Test it continuously: (will extract the project then run tests inside extracted project)
+# Test it continuously: (will extract the project then run tests inside extracted project as well as checking code formatting)
 sbt ~test
 
 # Change it:
 git checkout -b cool-feature-branch
-echo 'Sam sucks' >> README.md
-git commit -am 'Stating the evident'
+echo 'Cool feature' >> README.md
+git commit -am 'Added something!'
```

Comment:
I was thinking about updating these same lines, but didn't wanna let it go to your head 😂 glad you updated it

Reply:
Yeah, I figured there's no reason to state the obvious.  I was kind of flouting Grice's maxim of quantity here, and that's a bit of an antipattern.